config/jobs/kubernetes/sig-cloud-provider/gcp/gce-conformance.yaml

```diff
@@ -1,4 +1,41 @@
 periodics:
+- interval: 3h
+  cluster: k8s-infra-prow-build
+  name: ci-kubernetes-gce-conformance-latest
+  annotations:
+    fork-per-release: "true"
+    fork-per-release-replacements: "--extract=ci/latest-fast -> --extract=ci/latest-{{.Version}}"
+    testgrid-dashboards: conformance-all, conformance-gce
+    testgrid-tab-name: Conformance - GCE - master
+    description: Runs conformance tests using kubetest against kubernetes master on GCE
+  labels:
+    preset-service-account: "true"
+    preset-k8s-ssh: "true"
+  decorate: true
+  decoration_config:
+    timeout: 220m
+  spec:
+    containers:
+    - command:
+        - runner.sh
+        - /workspace/scenarios/kubernetes_e2e.py
+      args:
+      - --extract=ci/latest-fast
+      - --extract-ci-bucket=k8s-release-dev
+      - --gcp-master-image=gci
+      - --gcp-node-image=gci
+      - --gcp-zone=us-west1-b
+      - --provider=gce
+      - --test_args=--ginkgo.focus=\[Conformance\]
```

Comment:
i think you need to specify the `test-package-*` parameters too!

https://cs.k8s.io/?q=%5C-%5C-test%5C-package%5C-&i=nope&files=&excludeFiles=&repos=

Reply:
those options look to be for kubetest2, but this job is for kubetest v1 (so we have the original logs that apisnoop is looking for)